src/VisualStudio/Xaml/Impl/Implementation/LanguageServer/Handler/OnTypeRename/OnTypeRenameHandler.cs

```diff
@@ -7,7 +7,6 @@
 using System.Linq;
 using System.Threading;
 using System.Threading.Tasks;
-using Microsoft.CodeAnalysis.Editor.Xaml;
```

Comment:
Seems like a lot of the diff here is just cleaning up usings in files that weren't otherwise edited? If there's another commit could we omit those for ease of review?

Reply:
I can undo a lot of these if desired, they just came as part of the original changes that I've since reverted. I consider these good changes so I kept them, but I can remove them if you think simplifying the PR is more important.

Reply:
Went ahead and undid them